.github/workflows/update-browser-version.yml

```diff
@@ -4,8 +4,8 @@ name: 'Update the pinned browser version'
 
 on:
   schedule:
-    # Run hourly at xx:30.
-    - cron: '30 * * * *'
+    # Run every Monday at midnight and noon: https://crontab.guru/#0_0,12_*_*_1.
+    - cron: '0 0,12 * * 1'
```

Comment:
Are you sure we need to run it on Monday 2 times?

Reply:
I put this just as a fail-safe in case the first run fails, but it's not needed.

Let me revert it then.

Reply:
Done.